bundle/manifests/saas-operator-manager-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,256 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
After agreed on slack, I thought that this PR was going to add OLM, and update operator to be namespace scope, but it seems it keeps being cluster scope?

Reply:
ClusterScoped option removed in https://github.com/3scale/saas-operator/pull/52/commits/c91351f95b1f79890c900f0ed8613f1e1eb3012f